### PR TITLE
Create label for existing node

### PR DESCRIPTION
**Pull request by Priyankakore21**

### CODE REVIEW

File: src/browser/modules/EditorInfo/AddLabel.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  return (
+    <React.Fragment>
+      <StyledFavFolderButtonSpan>
+        <IconButton onClick={() => handleToggle(!textField)}>
+          <PlusIcon />
+        </IconButton>
+      </StyledFavFolderButtonSpan>
+      {textField ? (
+        <DrawerSection>
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddLabel.jsx]: why we need section and section body here

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 2]
```diff
@@ -48,7 +49,13 @@ const LabelSection = props => {
   return (
     <div>
       <DrawerSection>
-        <DrawerSubHeader>Labels</DrawerSubHeader>
+        <DrawerSubHeader>
+          Labels
+          <AddLabel
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx:54]: addLabel component inside sub header?

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 3]
```diff
@@ -89,9 +89,7 @@ export const handleFetchDataEpic = (action$, store) =>
         return noop
       })
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js:90]: now this file is hard to review cause of formatting changes. be on the same formatting

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx (hunk 2 above)
Comment (Priyankakore21) [reply src/browser/modules/EditorInfo/DisplayNodeDetails.jsx:54]: Yes, because it is the part of the sub header.

**Review by rahul-tarafdar-synerzip — APPROVED** [review]
LGTM

**Pull request merged**